docs/pages/image-extractor.rst

```diff
@@ -4,31 +4,82 @@ Habitat Sim Image Extractor Tutorial
 .. contents::
     :class: m-block m-default
 
+In this tutorial we will learn how to use the image extraction API in Habitat Sim by working through an example use case - training a semantic segmentation model. The tutorial will be structured as follows. We will first provide an overview of how to use the image extraction API. Next we will show how to use the API to train a semantic segmentation model. Finally, we will show how to define a custom pose extraction method, which is how a user programmatically defines the camera poses for image extraction.
 
-This notebook will go over how to use the image extraction API in Habitat Sim and the different
-user options available. It will also over how to create a custom pose extractor, which is how
-the image extraction API programmatically defines camera poses for the images.
+`Overview of Image Extraction`_
+===============================
 
-.. code:: py
+Habitat Sim provides an API to extract static images from a scene. The main class that handles image data extraction in Habitat Sim is called ImageExtractor. The user only needs to provide the scene filepath (either a .glb or .ply file).
 
-    import os
+``class ImageExtractor(scene_filepath, img_size, output, pose_extractor_name, shuffle, split, use_caching, pixels_per_meter)``
+
+**Args**
+
+* scene_filepath (required): A string or list of strings that is the filepath(s) to the scene file(s).
+* img_size: A tuple of (height, width) that represent the desired output image size. Default (512, 512).
+* output: A list of output images types. Options include any combination of ["rgba", "depth", "semantic"]. Default ["rgba"].
+* pose_extractor_name: The name of the pose extractor used to programmatically define camera poses for image extraction. If the user registered a custom pose extractor (see "Custom Pose Extraction" section), this is the name given during registration. Default "closest_point_extractor".
+* shuffle: Whether to shuffle the extracted images once they have been extracted. Default True.
+* split: A tuple of train/test split percentages. Must add to 100. Default (70, 30).
```

Comment:
Because a user cannot instantiate more than one ImageExtractor at a time, the only way I can see for them to separate their test/train sets is to keep track of which indices belong to which subset. This can still be done by specifying the mode as "full". It does simplify things a bit to only keep track of one "set" of images and have the user worry about the rest though; I could pursue this on a separate PR.